src/EVM/Solvers.hs

```diff
@@ -231,7 +228,7 @@ solverArgs solver timeout = case solver of
     [ "-in" ]
   CVC5 ->
     [ "--lang=smt"
-    , "--no-interactive"
+    -- , "--no-interactive"
```

Comment:
can we remove this instead of commenting it?

Reply:
done!